optaplanner-core/src/main/java/org/optaplanner/core/impl/localsearch/decider/acceptor/greatdeluge/GreatDelugeAcceptor.java

```diff
@@ -0,0 +1,87 @@
+package org.optaplanner.core.impl.localsearch.decider.acceptor.greatdeluge;
+
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.localsearch.decider.acceptor.AbstractAcceptor;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchMoveScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchPhaseScope;
+import org.optaplanner.core.impl.localsearch.scope.LocalSearchStepScope;
+import org.optaplanner.core.impl.score.ScoreUtils;
+
+public class GreatDelugeAcceptor extends AbstractAcceptor {
+
+    private Score initialWaterLevels = null;
+    private Score currentWaterLevel = null;
+
+    private Double rainSpeedRatio = DEFAULT_RAIN_SPEED_RATIO;
+    private Score rainSpeedScore = null;
```

Comment:
naming: rainSpeedScore is addend part of addition. Let's call it rainSpeedIncrement.

Reply:
Nitpick: put rainSpeedIncrement before rainSpeedRatio, because the stepEnded method handles rainSpeedIncrement before handling rainSpeedRatio

Reply:
Lukas meeting decision:
- waterLevelIncrementScore
- waterLevelIncrementRatio